brax/io/html.py

```diff
@@ -166,6 +166,52 @@ def render(sys: brax.System, qps: List[brax.QP]) -> str:
   return mesh;
 }
 
+function createHeightMap(heightMap) {
```

Comment:
Maintaining these js files inside python was difficult, so we moved them out to a `js/` directory - apologies for that change conflicting with yours.  Can you move these changes to the new location?

Reply:
Yes, moving to the new location is quite easy. 
But it now seems that testing and implementing new features on local branches might become very complicated due to how the js script is loaded : io/html.py makes a static reference to `https://cdn.jsdelivr.net/gh/google/brax@a93dadc48a41177a4d9e7794ce17c1ad98ed8583/js/viewer.js` which I believe is not easily editable by us.

Reply:
Yes agreed, local dev is not obvious how to do.  That static reference is not available to you, but if you fork the code, you can get your own cdn url using cdnjs, and you can use `@latest` in the url so that the url remains static.  That's still a bit annoying though because you have to push each time you want to try a new version of your js.

So here's another option for quick local dev:

In a cell in the colab, start up a local webserver that is pointed at your local checkout of the brax repo:

```
import threading
import socket
import http.server
import socketserver

class V6Server(socketserver.TCPServer):
  address_family = socket.AF_INET6

class Handler(http.server.SimpleHTTPRequestHandler):
  def __init__(self, *args, **kwargs):
    super().__init__(*args, directory='/path/to/brax', **kwargs)

port = 32888

def server_entry():
    httpd = V6Server(('::', port), Handler)
    httpd.serve_forever()

thread = threading.Thread(target=server_entry)
thread.start()
```

Then in `html.io`, change the javascript url like so:

![image](https://user-images.githubusercontent.com/45605/129980307-5f36406d-f89a-4cfd-ab79-6ede27175e20.png)

If this hack works for you, maybe we can introduce some kind of flag into the code to support this local dev more easily.

Reply:
No, it did not work for me. Although I can access it easily from the browser, the javascript import fails silently when I try to import the local file viewer.js. I will revert to the old method for dev, But this is really frustrating for local dev and for future contributors.

Reply:
Ahh, that's a bummer that you found this new js setup frustrating to dev.  Let's fix this!  Here's a colab that demonstrates serving local js:

https://colab.sandbox.google.com/gist/erikfrey/eb8b138dce06f7c2135e9d9690a09a8e/local_dev.ipynb#scrollTo=NaJDZqhCLovU

It might be a bit different if you were actually hosting the jupyter notebook on your local machine.  I just created an issue to track this here: https://github.com/google/brax/issues/33